roles/node_images/files/scripts/common/openscap.sh

```diff
@@ -24,24 +24,31 @@
 #
 set -eu
 
-SLES_MAJOR=$(awk -F= '/VERSION_ID/{gsub(/["]/,"");printf("%d", $NF)}' /etc/os-release)
-TEMP_DIR=$(mktemp -d)
+if [ "$RUN_OSCAP" = true ] ; then
```

Comment:
Maybe unindent everything, and just exit early?

```
if [ "$RUN_OSCAP" != true ]; then
  # touch files
  exit 0
fi
```

Reply:
That's brilliant